core/sql/executor/ExHdfsScan.cpp

```diff
@@ -631,7 +642,7 @@ ExWorkProcRetcode ExHdfsScanTcb::work()
                 ExRaiseSqlError(getHeap(), &diagsArea, 
                                 (ExeErrorCode)(EXE_ERROR_FROM_LOB_INTERFACE), NULL, 
                                 &intParam1, 
-                                &cliError, 
+                                &errno, 
```

Comment:
Again, not a safe usage; errno might have changed by this time (for example, underneath a getHeap() call).